### PR TITLE
feat(example): plumbing, intra-node routing for #308

### PR DESCRIPTION
**Pull request by petermetz**
As per the change request: https://github.com/hyperledger/cactus/pull/308#issuecomment-720195413

> Moreover, I don't feel comfortable because the title of your PR is "docs" and this PR includes not only documents but also actual codes. Would you mind if you divide your PR into docs(example) and feat(example)?

cc: @takeutak

### CODE REVIEW

**Review by jonathan-m-hamilton — APPROVED** [review]
LGTM

**Comment by takeutak** [comment]
What are the codes to change this pull-request?  Are they all 166 codes on "Files changed" or the part filtered with some commit name?

File: changelog.config.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    "breaking",
+    "issues",
+    "lerna",
+  ],
+  scopes: [],
+  types: {
+    build: {
+      description:
+        "Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)",
+      emoji: "🏗",
```
Comment (takeutak) [inline changelog.config.js:33]: These all emojis on this file are very confusing to me.  Would you mind if you delete these emojis?

**Review by takeutak — COMMENTED** [review]
@petermetz I almost agree to these pull-request. But I'm wondering the config of emojis on changelog.config.js. Would you mind if you delete these config of emojis?

**Comment by takeutak** [comment]
@petermetz Moreover, as above, I think there will be no problem except the changelog setting. However, from the next time, when issuing multiple Pull requests, please do not create Pull requests that depend heavily on other Pull requests, such as by separating branches. To be honest, it is very difficult to review.

File: changelog.config.js (hunk 1 above)
Comment (petermetz) [reply changelog.config.js]: @takeutak Done: https://github.com/hyperledger/cactus/pull/337/commits/30ad1b42c5a749b62f7eb26f74c6796d9e39e8f8

**Comment by petermetz** [comment]
> @petermetz I almost agree to these pull-request. But I'm wondering the config of emojis on changelog.config.js. Would you mind if you delete these config of emojis?

@takeutak Done: 30ad1b4

**Comment by petermetz** [comment]
> However, from the next time, when issuing multiple Pull requests, please do not create Pull requests that depend heavily on other Pull requests, such as by separating branches.

@takeutak If we fix the root cause, then I can stop submitting dependent PRs, yes, but until then this is the only solution I could come up with so far. (always open to alternative solutions of course)
Root cause: The average PR review time is 2 to 3 weeks even for PRs where I only change a few lines of code. Without the dependent PRs, I would be forced to spend 80% of my time idling or just spend the same time rebasing+conflict resolving dozens of branches daily.

> To be honest, it is very difficult to review.

@takeutak I hear you. Could you please point out which step is the difficult one from the step by step guides I provided in the past? I'm happy to expend on said steps, go deeper into specific parts of it if you give me some pointers.

Also happy to do pair programming style reviews where we do screen sharing and I assist with performing the reviews via the either the GitHub UI (or other preferred tool such as VSCode)

**Comment by takeutak** [comment]
LGTM

**Review by takeutak — APPROVED** [review]
LGTM.
@petermetz Please make the same correction for other dependent Pull requests. Otherwise, when merging dependent pull-requests, the modifications will be reversed by mistake.

**Comment by petermetz** [comment]
> LGTM.

@takeutak Cheers!

> @petermetz Please make the same correction for other your dependent pull requests before they are merged. Otherwise, when merging dependent pull-requests, the modifications will be reversed by mistake.

@takeutak Yeah, thankfully, rebase takes care of that automatically (assuming no conflicts) It's a big part of the reason why I strongly prefer rebase to merges in general.

**Pull request merged**